core/sql/sqlcomp/CmpSeabaseDDLupgrade.cpp

```diff
@@ -1739,7 +1799,31 @@ short CmpSeabaseMDupgrade::executeSeabaseMDupgrade(CmpDDLwithStatusInfo *mdui,
               }
 	  }
 	  break;
-
+	case OLD_LIBRARIES_DROP:
+	  {
+            if (upgradeLibraries.needsUpgrade(this))
+              {
+                if (upgradeLibraries.doDrops(&cliInterface,mdui,this))
+                  {
+                    // no status message in this case so no return
+                    cliInterface.clearGlobalDiags();
+                    mdui->setStep(OLD_REPOS_DROP);
+                    mdui->setSubstep(0);
+                    mdui->setEndStep(TRUE);
```

Comment:
In this step, we fail to drop the old libraries.  Do we just ignore any errors?

Reply:
Yes all the drop steps ignore errors.